projectOne/src/oneFive.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Arrays;
+import java.util.Scanner;
+public class oneFive {
+    public static void main(String[] args) {
+            Scanner scanner = new Scanner(System.in);
+            String name = "константин";
+            String lastName = "прадед";
+            String nameArray[] = {"к","о","н","с","т","а","н","т","и","н"};
+            String nameArrayEmpty[] = {"*","*","*","*","*","*","*","*","*","*"};
+            String lastNameArray[] = {"п","р","а","д","е","д"};
+            String lastNameEmptyArray[] = {"*","*","*","*","*","*"};
+            String usedLettersArray [] = new String [100];
```

Comment:
I don't fully understand why do you use different variables for the first and second names, it makes your code unreadable and long.